pallets/settlement-groth16/src/tests.rs

```diff
@@ -0,0 +1,59 @@
+use crate::mock;
+use crate::mock::*;
```

Comment:
just the second one is enought

Reply:
```suggestion
use crate::mock::*;
use crate::weight::WeightInfo;
```

Reply:
Done in https://github.com/HorizenLabs/zkVerify/pull/66/commits/484bf5bc5313c1a7f529be45c20272069b00249e